src/VirtoCommerce.Platform.Core/ProcessSettings/WkHtmlToPdfSettings.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+
+namespace VirtoCommerce.Platform.Core.ProcessSettings
+{
+    public class WkHtmlToPdfSettings : ProcessSettings
```

Comment:
I think this setting must be moved from there into modules where it is used.  For test purposes, you can define typed settings for a required command directly in the test solution.